exec/cnex/util.c

```diff
@@ -28,3 +28,4 @@ void fatal_error(const char *msg, ...)
     va_end(args);
     exit(1);
 }
+
```

Comment:
What is this extra blank line for? No need to make changes to files that don't have any meaning.